doc/sphinxman/source/cppe.rst

```diff
@@ -0,0 +1,213 @@
+.. #
+.. # @BEGIN LICENSE
+.. #
+.. # Psi4: an open-source quantum chemistry software package
+.. #
+.. # Copyright (c) 2007-2019 The Psi4 Developers.
+.. #
+.. # The copyrights for code used from other parties are included in
+.. # the corresponding files.
+.. #
+.. # This file is part of Psi4.
+.. #
+.. # Psi4 is free software; you can redistribute it and/or modify
+.. # it under the terms of the GNU Lesser General Public License as published by
+.. # the Free Software Foundation, version 3.
+.. #
+.. # Psi4 is distributed in the hope that it will be useful,
+.. # but WITHOUT ANY WARRANTY; without even the implied warranty of
+.. # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.. # GNU Lesser General Public License for more details.
+.. #
+.. # You should have received a copy of the GNU Lesser General Public License along
+.. # with Psi4; if not, write to the Free Software Foundation, Inc.,
+.. # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+.. #
+.. # @END LICENSE
+.. #
+
+.. include:: autodoc_abbr_options_c.rst
+
+.. index:: CPPE, PE
+
+.. _`sec:cppe`:
+
+Interface to CPPE by M. Scheurer
+=======================================
+
+.. codeauthor:: Maximilian Scheurer
+.. sectionauthor:: Maximilian Scheurer
+
+*Module:* :ref:`Keywords <apdx:cppe>`, :ref:`PSI Variables <apdx:cppe_psivar>`
+
+.. image:: https://img.shields.io/badge/home-cppe-informational.svg
+   :target: https://github.com/maxscheurer/cppe
+
+.. .. raw:: html
+.. 
+..    <br>
+.. 
+.. .. image:: https://img.shields.io/badge/docs-latest-5077AB.svg
+..    :target: http://pcmsolver.readthedocs.io/en/latest/
+
+|PSIfour| contains code to interface to the CPPE library developed
+by M. Scheurer.
+The CPPE library requires no additional licence, downloads, or
+configuration. The library allows for calculations in solution with the
+polarizable embedding model (PE), an explicit, fragment-based solvent model [Olsen:2010:3721]_.
+
+For a general tutorial on how to prepare/perform PE calculations, read the
+`tutorial review <https://onlinelibrary.wiley.com/doi/full/10.1002/qua.25717>`_.
+
+Installation
+~~~~~~~~~~~~
+
+**Binary**
+
+* .. image:: https://anaconda.org/psi4/cppe/badges/version.svg
+     :target: https://anaconda.org/psi4/cppe
+
+* CPPE is available as a conda package for Linux and macOS (and Windows, through the Ubuntu shell).
```

Comment:
Any chance of this compiling on native Windows? We can likely help with that and we might have some appveyor/Azure tips for you for the CI.

Reply:
I guess it should compile on native Windows since `cppe` ships with Q-Chem, and Q-Chem is available for Windows... though I have no idea how to setup the CI for Windows. Help would be much appreciated, @dgasmith, probably in a PR for `cppe`?

Reply:
A good place to start might be [here](https://github.com/MolSSI/cookiecutter-cms/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/appveyor.yml) which is the MolSSI cookie cutter example. This does not include compiled code, but will get you setup with Appveyor. Another example is [gau2grid](https://github.com/dgasmith/gau2grid/blob/master/appveyor.yml) which has some compiled code. The two together should give you enough information. If you can get a PR together I can likely comment on it on CPPE git.